docs/elementaloperatorchart-reference.md

```diff
@@ -17,15 +17,15 @@ The associated chart bootstraps an elemental-operator deployment on the [Rancher
 ## Get Helm chart info
 
 ```console showLineNumbers
-helm pull oci://registry.opensuse.org/isv/rancher/elemental/stable/charts/rancher/elemental-operator-chart
-helm show all oci://registry.opensuse.org/isv/rancher/elemental/stable/charts/rancher/elemental-operator-chart
+helm pull oci://registry.suse.com/rancher/elemental-operator-chart
+helm show all oci://registry.suse.com/rancher/elemental-operator-chart
 ```
 
 ## Install Chart
 
 ```console showLineNumbers
 helm install --create-namespace -n cattle-elemental-system elemental-operator \
```

Comment:
Here (and few lines above) we have not added the crds chart... probably we have to in another issue/PR as this whole docs section should be reviewed